home/tests.py

```diff
@@ -205,3 +205,41 @@ def tags_response(self, client):
         url = reverse('tags')
         response = client.get(url)
         return response
+
+
+class TestExplorePage:
+    class TestModelFunctions:
+
+        @pytest.mark.parametrize("question_id, expected_result", [
+            ("1", 5),
+            ("2", 1),
+            ("3", 0),
+            ("4", 0),
+        ])
+        @pytest.mark.django_db
+        def test_answers_num_func(self, question_id, expected_result):
+            question = Question.objects.get(id=question_id)
+            answers_num = question.get_answers_num()
+            assert answers_num == expected_result
+
+    class TestExplorePageView:
+
+        @pytest.mark.django_db
+        @pytest.fixture
+        def explore_page_response(self, client):
+            url = reverse('explore-page')
+            response = client.get(url)
+            return response
+
```

Comment:
all the tests below just inspect different aspects of the same response you get from doing a plain request for the page - they can probably be together in the same test.